src/libs/actions/IOU.js

```diff
@@ -152,6 +152,19 @@ function rejectTransaction({
         });
 }
 
+/**
+ * Sets IOU'S selected currency
+ *
+ * @param {Object} params.selectedCurrencyCode
+ * @param {Object} params.selectedCurrencySymbol
+ */
+function setSelectedCurrency({selectedCurrencyCode, selectedCurrencySymbol}) {
+    Onyx.merge(ONYXKEYS.MY_PERSONAL_DETAILS, {
+        selectedCurrencyCode,
+        selectedCurrencySymbol,
```

Comment:
Don't all codes have a corresponding symbol? 
Why store both when you could use one to infer the other by using a map of currencies? 

e.g.

```json
{
    "USD": "$",
    "EUR": "€"
}
```

Reply:
Done!